packages/console/src/components/Pagination/index.module.scss

```diff
@@ -0,0 +1,37 @@
+@use '@/scss/underscore' as _;
+
+.pagination {
+  display: flex;
+  justify-content: right;
+
+  li {
+    list-style: none;
+
+    &:not(:first-child) {
+      margin-left: _.unit(2);
+    }
+
+    .button {
+      border-radius: _.unit(1.5);
+      padding: 0;
+      width: 32px;
+      height: 32px;
+      text-overflow: unset;
+
+      > span {
+        margin: 0 auto;
+      }
+    }
+
+    &.disabled {
+      .button {
+        cursor: not-allowed;
+        background: var(--color-neutral-95);
```

Comment:
It seems like disabled button also has a 0.5 opacity

Reply:
Same as above, treat "Design System" as our standard.